scripts/run-kani.sh

```
@@ -0,0 +1,33 @@
+# kani-compiler is dynamically linked against the toolchain's libLLVM.
+# On macOS use DYLD_LIBRARY_PATH, on Linux use LD_LIBRARY_PATH.
+if [ "$(uname -s)" = "Darwin" ]; then
+    export DYLD_LIBRARY_PATH="${KANI_INSTALL_DIR}/toolchain/lib${DYLD_LIBRARY_PATH:+:${DYLD_LIBRARY_PATH}}"
+else
+    export LD_LIBRARY_PATH="${KANI_INSTALL_DIR}/toolchain/lib${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}"
+fi
+# CBMC tools (goto-cc, cbmc, goto-instrument) and the matching nightly
+# toolchain (cargo, rustc) must be reachable.
+export PATH="${KANI_INSTALL_DIR}/bin:${KANI_INSTALL_DIR}/toolchain/bin:${PATH}"
+# Tell rustup to use the Kani-pinned toolchain.
+export RUSTUP_TOOLCHAIN=$(cat "${KANI_INSTALL_DIR}/rust-toolchain-version")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)